src/CONST.js

```diff
@@ -46,6 +46,16 @@ const CONST = {
         RESERVED_FIRST_NAMES: ['Expensify', 'Concierge'],
     },
 
+    CALENDAR_PICKER: {
+        MIN_YEAR: 1900,
+        MAX_YEAR: 2038,
+    },
+
+    DATE_BIRTH: {
+        MIN_AGE: 5,
```

Comment:
Same here, why 5?

Reply:
These values come from DateOfBirthPage in validate functions, have been moved to const